### PR TITLE
#11:Feature/5 responsive header for tablet and phone

### PR DESCRIPTION
**Pull request by PhamNhuQuocBao**
image
image

-----
image
image

-----
image
image

### CODE REVIEW

File: src/layouts/header.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<header>
+  <div class="container">
+    <div class="header_top">
+      <p class="header_top-logo">
+        hofmann
+      </p>
+      <nav class="header_top-navigation">
+        <ul>
+          <li class="item"><a href="">mission</a></li>
```
Comment (haunguyen15-tt) [inline src/layouts/header.html]: Can you use the `each` tag here?

File: src/layouts/header.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      <div class="header_bottom-main">
+        <div class="content">
+          <p class="title">Choosing The Right Boutique Hotel For You</p>
+          <p class="subtitle">How To Fly A Private Jet On Your Next Trip</p>
+          <form action="" method="post">
+            <input type="email" name="email" id="email" class="input-email" placeholder="Your Email">
+            <button type="submit" class="btn btn-submit">find now</button>
+          </form>
+          <div class="icon">
+            <div class="icon-plane">
```
Comment (haunguyen15-tt) [inline src/layouts/header.html]: same

File: src/stylesheets/layouts/_header.css [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+  border-radius: 50%;
+  background-color: var(--bg-color);
+  margin: 0 30px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+/* responsive for tablet */
+@media screen and (max-width: 818px) and (min-width: 542px) {
```
Comment (haunguyen15-tt) [inline src/stylesheets/layouts/_header.css:138]: Can you create a mixin like this https://github.com/TITO-TRAINING/practice-html-css/blob/main/src/stylesheets/abstracts/_mixins.scss

**Comment by haunguyen15-tt** [comment]
Can you use `SASS` instead of `CSS`

**Comment by PhamNhuQuocBao** [comment]
@demons1502 I used CSS more than 50% in this project. If I change, it can be difficult for me. I will learn about SASS and apply for next project (if possible).

File: src/stylesheets/layouts/_header.css [hunk 4]
```diff
@@ -0,0 +1,272 @@
+@import url('../_reset.css');
+
+.header_top {
+  width: 100%;
+  height: 156px;
```
Comment (Lanng21) [inline src/stylesheets/layouts/_header.css]: Limit using sizes over 100px

File: src/stylesheets/layouts/_header.css [hunk 5]
```diff
@@ -0,0 +1,272 @@
...
+
+.header_top-navigation {
+  display: flex;
+  align-items: center;
+}
+
+.header_top-navigation a {
+  text-transform: uppercase;
+  font-size: 18px;
+  color: #ffffff;
```
Comment (Lanng21) [inline src/stylesheets/layouts/_header.css]: define const for this color

File: src/stylesheets/layouts/_header.css [hunk 6]
```diff
@@ -0,0 +1,272 @@
...
+}
+
+ul .item {
+  margin: 0 22px;
+  text-align: center;
+}
+
+.item:last-child {
+  padding: 15px 25px;
+  border: 2px solid #ffffff;
```
Comment (Lanng21) [inline src/stylesheets/layouts/_header.css]: the same

File: src/stylesheets/layouts/_header.css [hunk 7]
```diff
@@ -0,0 +1,272 @@
...
+  right: 0;
+  left: 0;
+  bottom: 0;
+  background-color: var(--text-color-primary);
+  opacity: 0.3;
+}
+
+.header_bottom-main {
+  height: 768px;
+  /* background: url('../../assets/images/background.png'); */
```
Comment (Lanng21) [inline src/stylesheets/layouts/_header.css]: Remove unused code

File: src/stylesheets/layouts/_header.css [hunk 8]
```diff
@@ -0,0 +1,272 @@
...
+ul .item {
+  margin: 0 22px;
+  text-align: center;
+}
+
+.item:last-child {
+  padding: 15px 25px;
+  border: 2px solid #ffffff;
+  border-radius: 50px;
+  width: 180px;
```
Comment (Lanng21) [inline src/stylesheets/layouts/_header.css]: not  set the height and width here?

File: src/stylesheets/layouts/_header.css [hunk 9]
```diff
@@ -0,0 +1,272 @@
...
+}
+
+form {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 60px;
+}
+
+.input-email {
+  width: 270px;
```
Comment (Lanng21) [inline src/stylesheets/layouts/_header.css]: same

File: src/stylesheets/layouts/_header.css [hunk 10]
```diff
@@ -0,0 +1,272 @@
...
+  ul .item {
+    margin: 0 15px;
+  }
+
+  .header_top-navigation a {
+    font-size: 11px;
+  }
+
+  .item:last-child {
+    /* height: 40px; */
```
Comment (Lanng21) [inline src/stylesheets/layouts/_header.css:156]: remove

File: src/stylesheets/layouts/_header.css [hunk 11]
```diff
@@ -0,0 +1,272 @@
...
+    width: 148px;
```
Comment (Lanng21) [inline src/stylesheets/layouts/_header.css:157]: width

File: src/stylesheets/layouts/_header.css [hunk 12]
```diff
@@ -0,0 +1,272 @@
...
+    padding: 11px;
+    margin-left: 0;
+  }
+
+  .title {
+    font-size: 32px;
+    width: 405px;
```
Comment (Lanng21) [inline src/stylesheets/layouts/_header.css:164]: same

File: src/stylesheets/layouts/_header.css [hunk 13]
```diff
@@ -0,0 +1,272 @@
...
+  }
+
+  .subtitle {
+    font-size: 11px;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  .input-email {
+    width: 228px;
```
Comment (Lanng21) [inline src/stylesheets/layouts/_header.css:176]: the same

File: src/stylesheets/layouts/_header.css [hunk 14]
```diff
@@ -0,0 +1,272 @@
...
+    font-weight: 400;
+  }
+
+  .subtitle {
+    font-size: 11px;
+  }
+
+  form {
+    display: block;
+    width: 190px;
```
Comment (Lanng21) [inline src/stylesheets/layouts/_header.css:235]: same

**Pull request merged**